shardingsphere-features/shardingsphere-sharding/shardingsphere-sharding-core/src/main/java/org/apache/shardingsphere/sharding/algorithm/keygen/CosIdKeyGenerateAlgorithm.java

```diff
@@ -20,57 +20,47 @@
 import me.ahoo.cosid.CosId;
 import me.ahoo.cosid.provider.IdGeneratorProvider;
 import me.ahoo.cosid.provider.LazyIdGenerator;
-import org.apache.shardingsphere.infra.config.algorithm.ShardingSphereInstanceRequiredAlgorithm;
-import org.apache.shardingsphere.infra.instance.InstanceContext;
 import org.apache.shardingsphere.sharding.algorithm.sharding.cosid.CosIdAlgorithm;
 import org.apache.shardingsphere.sharding.spi.KeyGenerateAlgorithm;
 
-import java.util.Optional;
 import java.util.Properties;
 
 /**
  * CosId key generate algorithm.
  */
-public final class CosIdKeyGenerateAlgorithm implements KeyGenerateAlgorithm, ShardingSphereInstanceRequiredAlgorithm {
-    
+public final class CosIdKeyGenerateAlgorithm implements KeyGenerateAlgorithm {
+
     public static final String TYPE = CosId.COSID.toUpperCase();
-    
```

Comment:
@Ahoo-Wang Please keep indents consistent with the previous one.

Reply:
> @Ahoo-Wang Please keep indents consistent with the previous one.

@Ahoo-Wang Please modify these checkstyle problem.

Reply:
I'm sorry I didn't find the question about indentation.Do you mean the indentation of empty lines?
![image](https://user-images.githubusercontent.com/4384159/151279591-09cf2f14-a0c8-480e-bf37-28c6efffaae2.png)

Reply:
> @strongduanmu I'm sorry I didn't find the question about indentation.Do you mean the indentation of empty lines? ![image](https://user-images.githubusercontent.com/4384159/151279591-09cf2f14-a0c8-480e-bf37-28c6efffaae2.png)

@Ahoo-Wang Sorry, I was wrong, this is a requirement for `code conduct`——Keep indents consistent with the previous one.

Reply:
@strongduanmu Ok,thanks.